pkg/bus/meseeks.go

```diff
@@ -148,9 +148,11 @@ func (b *Bus) newSeekRequest(ctx context.Context, auth, userID, connID string,
 		if err != nil {
 			return err
 		}
-		err = checkForBlock(ctx, b, connID, requester, receiver)
+		blockingState, err := checkForBlock(ctx, b, connID, requester, receiver)
 		if err != nil {
 			return err
+		} else if blockingState == 0 || blockingState == 1 {
+			return nil
```

Comment:
@domino14 I'm not sure if I should return a new error here instead

Reply:
that's a good question. It might be better to leave it as is, just with no information about blocking.